cromwell-helm/templates/postgres.yaml

```diff
@@ -50,3 +64,44 @@ spec:
   ports:
     - port: {{ .Values.db.port }}
       targetPort: {{ .Values.db.port }}
+---
+####################################
+# Database setup as a startup script
+####################################
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-postgres-startup
+data:
+  {{ .Values.config.postgres.startup_scripts.create_dbs }}: |-
+    #!/bin/bash
+    # Copied from: https://github.com/mrts/docker-postgresql-multiple-databases/blob/master/create-multiple-postgresql-databases.sh
+
+    set -e
+    set -u
+
+    function create_user_and_database() {
+    	local database=$1
+    	echo "  Creating user and database '$database'"
+    	psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
+    	    CREATE USER $database;
+    	    CREATE DATABASE $database;
+    	    GRANT ALL PRIVILEGES ON DATABASE $database TO $database;
```

Comment:
I'm not sure I love the user and DB name being identical for clarity reasons, but not a blocker